webapp/src/common/types.ts

```diff
@@ -652,11 +560,11 @@ export interface AreaInfoDTO extends AreaCreationDTO {
 
 export interface TaskView {
   id: string;
-  name: ReactNode;
-  dateView: ReactNode;
-  action: ReactNode;
+  name: React.ReactNode;
+  dateView: React.ReactNode;
+  action: React.ReactNode;
   date: string;
-  type: TaskType;
+  type: TTaskType | "DOWNLOAD" | "LAUNCH" | "UNKNOWN";
   status: string;
 }
```

Comment:
I use `T` as prefix here because of the name of the enums, without them, I would name the type like I named the enum (without `Type` suffix).

`TaskType` and `TTaskType`
`TaskStatus` and `TTaskStatus`

vs

`Task` and `TaskType`
`TaskStatus` and `TaskStatusType`

The `T` prefix allows you to resolve this particular case without renaming the enumerations and make things complicated. And I don't like `Type` suffix personally.

See [https://www.totaltypescript.com/tips/how-to-name-your-types](https://www.totaltypescript.com/tips/how-to-name-your-types)

`O.UnionOf<typeof TaskStatus>` is the same as `(typeof TaskStatus)[keyof typeof TaskStatus]`